.config/nextest.toml

```diff
@@ -10,7 +10,7 @@ experimental = ["setup-scripts"]
 [[profile.default.scripts]]
 # Exclude omicron-dev tests from crdb-seed as we explicitly want to simulate an
 # environment where the seed file doesn't exist.
```

Comment:
Could you also expand this comment to cover `omicron-live-tests`?

Reply:
Done in c7cd0ed63.